api/routes/DoorCode.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const router = express.Router();
+const DoorCode = require('../models/DoorCode');
+const settings = require('../util/settings');
+const logger = require(`${settings.util}/logger`);
```

Comment:
we are about to delete `logger` and `settings`, i would remove the usage from this file